thorlcr/activities/fetch/thfetchslave.cpp

```diff
@@ -181,8 +179,8 @@ class CFetchStream : public IRowStream, implements IStopInput, implements IFetch
         keyOutStream.setown(distributor->connect(keyRowIf, keyIn, fposHash, NULL, NULL));
     }
     virtual IRowStream *queryOutput() override { return this; }
-    virtual IFileIO *getPartIO(unsigned part) override { assertex(part<files); return fPosMultiPartTable[part].file->getFileIO(); }
-    virtual StringBuffer &getPartName(unsigned part, StringBuffer &out) override { return getPartFilename(parts.item(part), fPosMultiPartTable[part].location, out, true); }
+    virtual IFileIO *getPartIO(unsigned part) override { assertex(part<files); return LINK(fPosMultiPartTable[part].file); }
+    virtual StringBuffer &getPartName(unsigned part, StringBuffer &out) override { return getPartFilename(parts.item(part), 0, out, true); }
```

Comment:
Why change second arg of getPartFilename() from fPosMultiPartTable[part].location to 0 ?

Reply:
@mckellyln - because it was never used (only in error), see my comment in PR I added before near top.